PaddleAudio/paddleaudio/models/wav2vec2/config_utils.py

```diff
@@ -0,0 +1,628 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors and The HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Configuration base class and utilities."""
+
+import copy
+import json
+import os
+from typing import Any, Dict, Tuple, Union
+
+from paddleaudio.utils.log import logger
+
+__all__ = ['PretrainedConfig']
+
+
+class PretrainedConfig(object):
```

Comment:
Try to simplify this config class.

Reply:
Because we just need a dictionary as config.

Reply:
done